src/main/kotlin/com/refinedmods/refinedstorage/gui/screen/ConstructorScreen.kt

```diff
@@ -0,0 +1,34 @@
+package com.refinedmods.refinedstorage.gui.screen
+
+import com.refinedmods.refinedstorage.RS
+import com.refinedmods.refinedstorage.gui.screenhandlers.ConstructorScreenHandler
+import com.refinedmods.refinedstorage.gui.widget.sidebutton.ConstructorDropSideButton
+import com.refinedmods.refinedstorage.gui.widget.sidebutton.ExactModeSideButton
+import com.refinedmods.refinedstorage.gui.widget.sidebutton.RedstoneModeSideButton
+import com.refinedmods.refinedstorage.gui.widget.sidebutton.TypeSideButton
+import com.refinedmods.refinedstorage.tile.ConstructorTile
+import com.refinedmods.refinedstorage.tile.NetworkNodeTile
+import net.minecraft.client.util.math.MatrixStack
+import net.minecraft.entity.player.PlayerEntity
+import net.minecraft.text.Text
+
+class ConstructorScreen(
+        gui: ConstructorScreenHandler,
+        player: PlayerEntity,
+        title: Text
+) : BaseScreen<ConstructorScreenHandler>(gui, player, title) {
+
+        override fun onPostInit(x: Int, y: Int) {
+                this.x -= 60
```

Comment:
Could you explain what this does?

Reply:
Test code that should be removed (Was trying to shift the gui over so that REI didn't overlap, but it doesn't help.)